dev/archery/archery/cli.py

```diff
@@ -744,6 +744,9 @@ def _set_default(opt, default):
 @click.option('--with-rust', type=bool, default=False,
               help='Include Rust in integration tests',
               envvar="ARCHERY_INTEGRATION_WITH_RUST")
+@click.option('--target-languages', default='',
+              help=('Target languages in this integration tests'),
+              envvar="ARCHERY_INTEGRATION_TARGET_LANGUAGES")
```

Comment:
Uh, can the help string be more explanatory? It's really not possible to understand what this does from the current wording.

Reply:
Sure. But is the help string a suitable location to explain it more with click API?
In general, `--help` output uses a simple one-line (or a few lines) per option.

Reply:
Thanks. I'll use it: GH-44158